README.md

```diff
@@ -45,6 +45,20 @@ If you haven't, create a superuser with the [Django createsuperuser command](htt
 - Create a new drip.
 - Select and delete drips.
 
+### Support SMS(Optional)
+- Make sure to have a field or property called `phone_number` in your user model, and its value is international `e.g. +12125552368`
```

Comment:
1. re: `phone_number`, the logic on this library doesn't depend on `phone_number` or how it's linked with the user so I don't think we should recommend doing it one way or another.

2. I think we need to provide a generic description about why sms sending logic is not implemented here and having an example as illustrated below is great